### ISSUE TITLE
Bug: Uncaught Exception for organization-wide CloudTrails

### ISSUE THREAD

**Issue by holgerjenczewski1007** [jantman/awslimitchecker#439]
## Bug Report

`awslimitchecker/services/cloudtrail.py` throws an exception when trying to get details about an organization-wide CloudTrail which originates from another AWS account (the organization root account).

### Version

awslimitchecker 7.1.0

### Installation Method

virtualenv

### Supporting Software Versions

Python 3.6.8
Virtualenv 16.6.0
 (on Ubuntu Linux 18.04)

### Actual Output

```
> awslimitchecker -r eu-central-1
[...]
Traceback (most recent call last):
  File "/<SOME_PATH>/.venv/bin/awslimitchecker", line 8, in <module>
    sys.exit(console_entry_point())
  File "/<SOME_PATH>/.venv/lib/python3.6/site-packages/awslimitchecker/runner.py", line 531, in console_entry_point
    r.console_entry_point()
  File "/<SOME_PATH>/.venv/lib/python3.6/site-packages/awslimitchecker/runner.py", line 503, in console_entry_point
    res, problems, problem_str = self.check_thresholds(metrics)
  File "/<SOME_PATH>/.venv/lib/python3.6/site-packages/awslimitchecker/runner.py", line 320, in check_thresholds
    service=self.service_name
  File "/<SOME_PATH>/.venv/lib/python3.6/site-packages/awslimitchecker/checker.py", line 554, in check_thresholds
    tmp = cls.check_thresholds()
  File "/<SOME_PATH>/.venv/lib/python3.6/site-packages/awslimitchecker/services/base.py", line 272, in check_thresholds
    self.find_usage()
  File "/<SOME_PATH>/.venv/lib/python3.6/site-packages/awslimitchecker/services/cloudtrail.py", line 67, in find_usage
    self._find_usage_cloudtrail()
  File "/<SOME_PATH>/.venv/lib/python3.6/site-packages/awslimitchecker/services/cloudtrail.py", line 90, in _find_usage_cloudtrail
    TrailName=trail['Name']
  File "/<SOME_PATH>/.venv/lib/python3.6/site-packages/botocore/client.py", line 357, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/<SOME_PATH>/.venv/lib/python3.6/site-packages/botocore/client.py", line 661, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.errorfactory.TrailNotFoundException: An error occurred (TrailNotFoundException) when calling the GetEventSelectors operation: Unknown trail: arn:aws:cloudtrail:eu-central-1:<CURRENT_AWS_ACCOUNT_ID>:trail/<TRAIL_NAME> for the user: <CURRENT_AWS_ACCOUNT_ID>
```
Exception is caused by an organization-wide CloudTrail, originating from another AWS account (organization root account).

"aws cloudtrail describe-trails" shows this trail as follows (note the different ARN):
```
        {
            "Name": "<TRAIL_NAME>",
            "S3BucketName": "<BUCKET_NAME_PREFIX>-global-cloudtrail",
            "IncludeGlobalServiceEvents": true,
            "IsMultiRegionTrail": true,
            "HomeRegion": "eu-central-1",
            "TrailARN": "arn:aws:cloudtrail:eu-central-1:<OTHER_AWS_ACCOUNT>:trail/<TRAIL_NAME>",
            "LogFileValidationEnabled": true,
            "CloudWatchLogsLogGroupArn": "arn:aws:logs:eu-central-1:<OTHER_AWS_ACCOUNT>:log-group:/aws/cloudtrail/global-trail-logs:*",
            "CloudWatchLogsRoleArn": "arn:aws:iam::<OTHER_AWS_ACCOUNT>:role/global-trail-resources-EcGlobalTrailLogRole-XXXXXXXXXXXX",
            "HasCustomEventSelectors": false,
            "IsOrganizationTrail": true
        }
```
Since the ARN gets clobbered in the call, this could potentially be a boto3 issue.

### Expected Output

(normal aws limits report)

**Comment by jantman** [comment]
@holgerjenczewski1007 Thanks for reporting this. I'm actually going to be doing some work this week for a major release, so I'll try to get this included.

**Comment by jantman** [comment]
@holgerjenczewski1007 I'm working on a fix for this now. It's definitely a bug in awslimitchecker.

We're looping over every CloudTrail that's returned by ``DescribeTrails``, and then calling ``GetEventSelectors`` on every one that is in our current region. ``GetEventSelectors`` is being called only with the trail name, not the ARN, and boto3 is constructing the ARN based on the current account.

It appears that there are 3 pieces to the fix:

1. Call ``GetEventSelectors`` with the full ARN, not just the name.
2. Call ``DescribeTrails`` with [includeShadowTrails](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudtrail.html#CloudTrail.Client.describe_trails) set to False, to exclude replications of trails in other regions or organization trails in member accounts.
3. If we can't get event selectors for a trail, just don't count them, instead of throwing an exception.

**Comment by jantman** [comment]
A fix for this has been merged to the ``develop`` branch in #443 and will be present in the next release. Thank you so much for reporting this, and apologies for the issue!

**Comment by jantman** [comment]
This has been fixed in [8.0.0](https://github.com/jantman/awslimitchecker/releases/tag/8.0.0) which is now live on [PyPI](https://pypi.org/project/awslimitchecker/) and [Docker Hub](https://hub.docker.com/r/jantman/awslimitchecker/tags). Apologies for any delays in getting this released.

**Comment by holgerjenczewski1007** [comment]
Great work. Thank you!
